cc/tpu_test.cc

```diff
@@ -0,0 +1,224 @@
+// Copyright 2018 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// Standalone test code that runs the Minigo model on a Cloud TPU.
+// Helpful for debugging issues.
+//
+// Example usage (you will need to supply your own values for tpu_name, model_a
+// and model_b):
+//   bazel build --define=tf=1 --define=tpu=1 cc:tpu_test
+//   ./bazel-bin/cc/tpu_test \
+//     --tpu_name=grpc://10.240.2.10:8470 \
+//     --model_a=gs://tmadams-sandbox/tpu_cpp/000674-neptune.pb \
```

Comment:
if you use the models in minigo-pub they'll be publicly available to any of the tf folks who happen by